src/Basque/Mews.Fiscalizations.Basque/DtoToModelConverter.cs

```diff
@@ -14,6 +14,23 @@ internal static SendInvoiceResponse Convert(
         string tbaiIdentifier,
         String1To100 signatureValue)
     {
+        if (response is null)
```

Comment:
I would prefer to have this somewhere else and not in this method, or at least make the response nullable in the parameters.